libraries/AP_GPS/GPS_Backend.cpp

```diff
@@ -169,3 +170,42 @@ void AP_GPS_Backend::Write_DataFlash_Log_Startup_messages() const
     _detection_message(buffer, sizeof(buffer));
     gps._DataFlash->Log_Write_Message(buffer);
 }
+
+
+void AP_GPS_Backend::send_mavlink_gps_rtk(mavlink_channel_t chan, uint8_t inst)
+{
+    // send status
+    if (inst == 0) {
+        mavlink_msg_gps_rtk_send(chan,
+                             rtk_state.time_last_baseline_ms,
+                             rtk_state.rtk_receiver_id,
+                             rtk_state.wn,
+                             rtk_state.tow,
+                             rtk_state.rtk_health,
+                             rtk_state.rtk_rate,
+                             rtk_state.nsats,
+                             rtk_state.baseline_coords_type,
+                             rtk_state.baseline_a_mm,
+                             rtk_state.baseline_b_mm,
+                             rtk_state.baseline_c_mm,
+                             rtk_state.accuracy,
+                             rtk_state.iar_num_hypotheses);
+    } else {
```

Comment:
This should ensure that the requested instance is instance 1. (I can see that at a future data someone might try and refactor this to just be for all gps backends call send gps rtk data)